apps/drupal/particle.theme

```diff
@@ -7,6 +7,16 @@
 
 use Drupal\Core\Form\FormStateInterface;
 
+/**
+ * Implements hook_preprocess_HOOK().
+ */
+function particle_preprocess_page(&$variables) {
+  $moduleHandler = \Drupal::service('module_handler');
+  if ($moduleHandler->moduleExists('components')){
+    $variables['components'] = TRUE;
```

Comment:
Oh man, a root page var named `components` is going to collide *real* quick. How about `has_components_module` or something Drupal standardized equivalent?